### PR TITLE
Tutorial Updates

### PR DESCRIPTION
**Pull request by abachma2**
Pulled in some feedback from the group, added DOI for input and output files. I have temporarily removed the DeployInst page from the tutorial for now, until I can make an input/output for that example too.

### CODE REVIEW

**Comment by abachma2** [comment]
I added the DeployInst input and output files in, so that part of the example is included back into the tutorial.

File: source/user/tutorial/add_arche.rst [hunk 1]
```diff
@@ -66,16 +66,15 @@ facility archetypes are:
 
 Activity: Discover the Available Archetypes
 ===========================================
+If using |Cyclus| on your machine, the archetypes available to you are only those that have been downloaded. 
+To check which archetypes are downloaded on your machine run the command ``cyclus -a`` from your terminal.
 
-Since archetypes can change without having to reinstall |Cyclus|, there is
-the ability to automatically discover which archetypes are available.
+If you are not ruuning |Cyclus| on your machine, the archetypes available to you include those in Cycamore, which
```
Comment (gwenchee) [inline source/user/tutorial/add_arche.rst:72]:
```suggestion
If you are not running |Cyclus| on your machine, the archetypes available to you include those in Cycamore, which 
```

**Review by gwenchee — APPROVED** [review]
Looks good to me!

**Comment by abachma2** [comment]
Can this be merged now @gwenchee Or do you want someone else to review it first?

**Pull request merged**